### PR TITLE
[no ticket] Add service method to test cors in doppler

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- Add service test method to test cors in doppler app
- Obtain user data from doppler and show in screen

### CODE REVIEW

**Comment by cbernat** [comment]
👍

File: src/components/Header/Nav.js [hunk 1]
```diff
@@ -3,11 +3,31 @@ import React, { Component } from 'react';
...
 export default class HeaderNav extends Component {
+  constructor() {
+    super();
+    this.state = {
+      data: {},
+    };
+  }
+
+  componentWillMount() {
+    fetch('http://localhost:52191/Reports/Reports/GetUserData', {
```
Comment (andresmoschini) [inline src/components/Header/Nav.js]: Should it be in the following way?

```suggestion
    return fetch('http://localhost:52191/Reports/Reports/GetUserData', {
```

Comment (andresmoschini) [inline src/components/Header/Nav.js]: I think that the related URL should not be _Reports_ but _WebApp_

File: src/components/Header/Nav.js [hunk 2]
```diff
@@ -3,11 +3,31 @@ import React, { Component } from 'react';
...
+      mode: 'cors',
+      credentials: 'include',
+    })
+      .then((response) => {
+        return response.json();
+      })
+      .then((data) => {
+        this.setState({ data: data.user });
+      });
```
Comment (andresmoschini) [inline src/components/Header/Nav.js]: Only curiosity, why did you use two _then_ in place of only one?

```js
      .then((response) => {
        const data = response.json();
        this.setState({ data: data.user });
      });
```

**Pull request merged**